ansible_runner/utils.py

```diff
@@ -78,6 +78,27 @@ def check_isolation_executable_installed(isolation_executable):
         return False
 
 
+def build_safe_env(env):
+    '''
+    Build environment dictionary, hiding potentially sensitive information
+    such as passwords or keys.
+    '''
+    HIDDEN_PASSWORD = '**********'
+    hidden_re = re.compile(r'API|TOKEN|KEY|SECRET|PASS', re.I)
+    urlpass_re = re.compile(r'^.*?://[^:]+:(.*?)@.*?$')
+    safe_env = dict(env)
+    for k, v in safe_env.items():
+        if k == 'AWS_ACCESS_KEY_ID':
```

Comment:
This is too specific to have in a generalized tool.

Reply:
This function is ported over almost directly as is from AWX.  We _could_ drop the unredacted env into the artifact file and then clean it up on the other side, if we are ok with that.

Reply:
I've gone ahead and done it that way now.  It seems like this isn't any worse than the existing situation, since we are dropping an envvars file with this stuff unredacted anyway.